docs/how_to_add_a_publisher.md

```diff
@@ -333,14 +335,140 @@ def title(self) -> Optional[str]:
 
 ### Extracting Attributes with XPath and CSS-Select
 
-Sometimes you have to get information directly from the DOM of the HTML (in most cases the article text).
-To do so we suggest using [**lxml**](https://lxml.de/) and selectors like [`XPath`](https://lxml.de/xpathxslt.html) or [`CSSSelect`](https://lxml.de/cssselect.html).
-You can use both selectors on the `doc` attribute of `Precomputed`.
-If you use a selector in any of your attributes (which will be most likely the case when extracting the actual article text) you should precompute the selector as a class variable using the `XPath` respectively `CSSSelector` classes of lxml.
+When parsing the `ArticleBody`, or in certain scenarios, you need to directly obtain information from the [Document Object Model](https://en.wikipedia.org/wiki/Document_Object_Model) (DOM) of the HTML/XML.
+The DOM serves as an interface representing the underlying HTML or XML file as a tree structure, where each element (tag) of the file functions as a node in the tree.
+To select or search respectively for the information you need you can access these nodes using selectors like CSS-Select or XPath.
+Fundus relies on the Python package `lxml` and its selector implementation.
+
+#### Working with `lxml`
+
+Consider the following HTML example.
+
+````html
+<html lang="de">
+    <head>
+        <meta charset="utf-8">
+        <title>...</title>
+    </head>
+    <body>
+        <h2>This is a heading.</h2>
+        <p>This is a paragraph inside the body.</p>
+        <p class="A">This is a paragraph with a class.</p>
+        <div>
+            <p>This is a paragraph within a div</p>    
+        </div>
+        <div class="B">
+            <p>This is a paragraph within a div of class B</p>    
+        </div>
+        <p additional-attribute="not allowed">This is a paragraph with a weird attribute</p>  
+    </body>
+</html>
+````
+
+To work with `lxml`  selectors, the initial step involves constructing an `Etree`, which represents the DOM of the HTML.
+This is achieved as follows:
+
+```` python
+root = lxml.html.document_fromstring(html)
+````
+
+This will return an object of type `lxml.html.HtmlElement` representing the root node of the DOM tree.
+Within the Fundus parser, the DOM tree is already generated for each article, and the root node can be accessed using the `doc` parameter of `Precomputed`.
```

Comment:
In my opinion, the explanation and the naming clashes. Why is the root node named doc? Why not root? It is probably best to slightly modify this paragraph.